<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>KylinCloud-I</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>M1</string>
    </property>
    <widget class="QPushButton" name="input_btn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>M2</string>
    </property>
    <widget class="QPushButton" name="extract_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>特征提取</string>
     </property>
    </widget>
    <widget class="QPushButton" name="match">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>匹配</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reconstruct">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>重建</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>M3</string>
    </property>
    <widget class="QPushButton" name="compute_laser">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>解算激光</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
